stratum/hal/lib/barefoot/bfrt_packetio_manager.h

```diff
@@ -135,12 +127,19 @@ class BfrtPacketioManager {
   size_t packetin_header_size_ GUARDED_BY(data_lock_);
   size_t packetout_header_size_ GUARDED_BY(data_lock_);
 
-  // Initialized to false, set once only on first PushForwardingPipelineConfig.
-  bool initialized_ GUARDED_BY(data_lock_);
+  //
+  std::shared_ptr<Channel<std::string>> packet_receive_channel_
+      GUARDED_BY(data_lock_);
+
+  //
+  pthread_t sde_rx_thread_id_ GUARDED_BY(data_lock_);
+
+  // Pointer to a BfSdeInterface implementation that wraps all the SDE calls.
+  BfSdeInterface* bf_sde_interface_ = nullptr;  // not owned by this class.
 
   // Fixed zero-based Tofino device number corresponding to the node/ASIC
   // managed by this class instance. Assigned in the class constructor.
-  const int device_id_;
+  const int device_;
```

Comment:
Any reason you want to remove the `_id` from the variable name? (also the parameter of constructor)

Reply:
More concise. For some types the `_id_` makes a difference: `node` (from common.proto) / `node_id`. Port is borderline. But for device I did not find a conflict, so I went for the same approach as with `unit`: https://github.com/stratum/stratum/search?q=unit_